docs/proposal/2022-06-14-bindings.md

```diff
@@ -0,0 +1,681 @@
+
+    </td>
+    <td>
+
+    ```yaml
+    notifiers:
+      - name: default
+        kubernetes:
+          # New 'namespace' property, allows Namespace restriction
+          # It can be overridden in nested level.
+          namespace:
+            include: ["@all"]
+          resources:
+            - name: v1/pods
+              namespaces: # override top level namespace
+                include:
+                  - istio-system
+              events:
+                - error
+            - name: networking.istio.io/v1alpha3/VirtualServices
+              # uses the default Namespace settings from top level definition.
+              events:
+                - error
+        # Recommendations about the
+        # best practices for the created resource
+        recommendations:
+          image:     # "Checks if 'latest' image tag is used for container image."
+            enabled: true
+          pod:       # "Checks if labels are missing in the pod specs."
+            enabled: true
+          ingress:   # "Checks if services and tls secrets used in ingress are available."
+            enabled: true
+    ```
+
+    </td>
+    </tr>
+    </table>
+
+
+2. Executors
+    <table>
+    <tr>
+    <td> Before </td> <td> After </td>
+    </tr>
+    <tr>
+    <td>
+
+    ```yaml
+    settings:
+      # Cluster name to differentiate incoming messages
+      clustername: not-configured
+      # Kubectl executor configs
+      kubectl:
+        enabled: false
+        commands:
+          verbs: ["api-resources", "...", "auth"]
+          resources: ["deployments", "...", "nodes"]
+    ```
+
+    </td>
+    <td>
+
+
+    ```yaml
+    executors:
+      - name: kubectl-read-only # name used for bindings
+        kubectl:
+          # New 'namespace' property, allows Namespace restriction
+          namespaces:
+            include:
+              - team-a
+          commands:
+            verbs: ["api-resources", "...", "auth"]
+            resources: ["deployments", "..", "nodes"]
+    ```
+
+    </td>
+    </tr>
+    </table>
+
+
+2. Communications
+    <table>
+    <tr>
+    <td> Before </td> <td> After </td>
+    </tr>
+    <tr>
+    <td>
+
+    ```yaml
+    communications:
+      # Settings for Slack
+      slack:
+        enabled: false
+        channel: 'SLACK_CHANNEL'
+        token: 'SLACK_API_TOKEN'
+
+
+
+
+
+
+
+    ```
+
+    </td>
+    <td>
+
+
+    ```yaml
+    communications:
+      - name: tenant-b-workspace
+        slack:
+          enabled: false
+          token: 'SLACK_API_TOKEN'
+          channels:
+            - name: "#team-a"
+              bindings:
+                notifiers:
+                  - "nodes-errors"
+                  - "depreacted-api"
+                executors:
+                  - "kubectl-read-only"
+                  - "helm-full-access"
+    ```
+
+    </td>
+    </tr>
+    </table>
+
+
+## Resources
+
+- [PR that adds profiles to BotKube configuration](https://github.com/infracloudio/botkube/pull/291)
```

Comment:
Maybe a small comment would be helpful - why it is good to mention it there 🙂  (e.g. first implementation, which was based on X, but it is not worth to invest in time, as it has limitations like Y)